lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -191,91 +203,112 @@ def run(self, **kwargs):
             f'json://{prvsnr_cluster_path}'
         )
 
-        if user != None and password != None and ip != None and port != None:
-            # fetch enclosure serial/id
-            self.enclosure_id = EnclosureInfo(ip, user, password, port).get_enclosure_serial()
-            self.mode = "primary"
 
-            self.refresh_conf_pillar_map()
+        setup_type = Conf.get (
+            f'server_node>{self.machine_id}>type'
+        )
 
-            self.logger.debug(f"Writing storage enclosure_id to the file {enc_file_path}")
-            file = open(enc_file_path, "w")
-            file.write(self.enclosure_id + "\n")
-            file.close()
+        if setup_type == "VM":
```

Comment:
Most of code after this can be dropped in favor of https://github.com/Seagate/cortx-prvsnr/blob/151c7d164ea4aae589a4a43fdc533da791883e0c/srv/components/system/storage/enclosure_id/config/set.sls

Reply:
I dont think we can drop the current code instead we can drop the set.sls file as whatever that module does is already handled during factory storage config